docs/source/release_notes/release-notes.rst

```diff
@@ -50,6 +50,11 @@ develop
     *    - Type
          - Change
 
+    *    - |new|
+         - ``TransactionManager``s now expose a ``TimestampManagementService``, allowing clients to fast-forward timestamps when necessary.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/3524>`__)
```

Comment:
Probably worth a bit of a warning, maybe along the lines of "this feature is only intended for libraries that directly extend AtlasDB functionality; it is very unlikely that AtlasDB users in general should directly use the timestamp management service"?